runOrthopairs.sh

```diff
@@ -6,11 +6,14 @@ cd $DIR
 
 ## Update repo
 git pull
-## Create new jar file with orthoinference code
+## Create new jar file with orthopairs code
 mvn clean compile assembly:single
 
+## Ensures the correct jar file is obtained regardless of orthopairs project version
+orthopairs_jar_file=$(ls target/orthopairs-*-jar-with-dependencies.jar)
```

Comment:
What happens if somehow there is more than one jar that matches the pattern in `target/`?

Reply:
That should never happen since this happens immediately after a running `mvn clean` and then re-compiling.  There will only be one file that has the pattern `orthopairs-*-jar-with-dependencies.jar`.

Reply:
Hmmm... if maven fails (to clean or compile or package), will the script abort or attempt to continue?

Reply:
It will abort after a failed command since it has `set -e` on line 2.